samples/SunburstChartExample/README.md

```diff
@@ -0,0 +1,23 @@
+# Issue 674 PPTX Sunburst chart? 
```

Comment:
No need to reference issue number - the more important thing is to fully describe the scenario